packages/flutter/lib/src/widgets/scroll_context.dart

```diff
@@ -41,6 +50,10 @@ abstract class ScrollContext {
   /// The direction in which the widget scrolls.
   AxisDirection get axisDirection;
 
+  /// Platform specific function applied to existing velocity during repeated
+  /// scrolls.
+  MomentumCarryFunction get momentumCarryFunction;
```

Comment:
this seems out of place compared to the other things on this class. It would be like having the overscroll indicator glow color here.